client/src/store/cards/cards.effects.ts

```diff
@@ -11,304 +11,46 @@ import { catchError, filter, map, switchMap, take, delay } from 'rxjs/operators'
-    skills: ['Teach a neural net', 'Automatically deals damage for 3 hp to enemy hero '],
-    hp: 4,
-    damage: 4,
-    manaCost: 4,
-    effects: {disableWhenSpellInUse: false}
-  },
-  {
-    _id: '151515',
-    class: 'CTO',
-    id: 14,
-    name: 'John',
-    surname: 'Stivenson',
-    image: 'http://simpleicon.com/wp-content/uploads/user1.png',
-    skills: [],
-    hp: 7,
-    damage: 6,
-    manaCost: 5,
-    effects: {disableWhenSpellInUse: false}
-  },
-  {
-    _id: '161616',
-    class: 'Spell',
-    id: 15,
-    name: 'Customer did not accept!!',
-    surname: '',
-    image: 'http://simpleicon.com/wp-content/uploads/user1.png',
-    skills: [],
-    hp: null,
-    damage: 6,
-    manaCost: 4,
-    effects: {disableWhenSpellInUse: false}
-  },
-  {
-    _id: '181818',
-    class: 'HR',
-    id: 17,
-    name: 'Ira',
-    surname: 'Chiz',
-    image: 'http://simpleicon.com/wp-content/uploads/user1.png',
-    skills: ['Teambuilding', 'Give all your minions at the table +1 atack'],
-    hp: 2,
-    damage: 2,
-    manaCost: 3,
-    effects: {disableWhenSpellInUse: false}
-  },
-  {
-    _id: '191919',
-    class: 'Spell',
-    id: 18,
-    name: 'Crush by experience',
-    surname: '',
-    image: 'http://simpleicon.com/wp-content/uploads/user1.png',
-    skills: ['Cast a spell','Turns any enemy minion to a ship with 1 hp and 1 attack'],
-    hp: null,
-    damage: null,
-    manaCost: 4,
-    effects: {disableWhenSpellInUse: false}
-  },
-  {
-    _id: '191919',
-    class: 'Spell',
-    id: 16,
-    name: 'Cruch by experience',
-    surname: '',
-    image: 'http://simpleicon.com/wp-content/uploads/user1.png',
-    skills: ['Cast a spell','Turns any enemy minion to a ship with 1 hp and 1 attack'],
-    hp: null,
-    damage: null,
-    manaCost: 4,
-    effects: {disableWhenSpellInUse: false}
-  },
+public myFirstMoveVariants = [true, false]   
 
-  {
-    _id: '202020',
-    class: 'Spell',
-    id: 19,
-    name: 'Budget agreed!',
-    surname: '',
-    image: 'http://simpleicon.com/wp-content/uploads/user1.png',
-    skills: ['Cast a spell','Gives you two additional cards from deck'],
-    hp: null,
-    damage: null,
-    manaCost: 3,
-    effects: {disableWhenSpellInUse: false}
-  },
 
-  {
-    _id: '212121',
-    class: 'Spell',
-    id: 20,
-    name: 'Budget agreed!',
-    surname: '',
-    image: 'http://simpleicon.com/wp-content/uploads/user1.png',
-    skills: ['Cast a spell','Gives you two additional cards from deck'],
-    hp: null,
-    damage: null,
-    manaCost: 3,
-    effects: {disableWhenSpellInUse: false}
-  }
 
-]
 
-public myFirstMoveVariants = [true, false]   //just a temporary mock 
+@Effect() public getCards$: Observable<Action> = this.actions$.pipe(
+  ofType<cardActions.LoadCards>(
+    cardActions.CardsActionTypes.LoadCards
+  ),
+  switchMap((action: cardActions.LoadCards) => {
+    let headers: HttpHeaders = new HttpHeaders();
+    headers = headers.append(
+      "JWT-Token",
```

Comment:
Create http interceptor that will set "JWT-Token" header in every request to our server.